.claude/hooks/gdd-permission-hook.sh

```
@@ -526,15 +526,20 @@ if [[ -f "$_rules_dir/hook-rules" ]]; then
+# Source the k8s guard for scoped-redirect evaluation (Tier 2b). Track whether
+# the policy loaded: Kubernetes-looking commands must force-ask if the guard is
+# missing or broken, rather than silently dropping the safety floor.
 # Use BASH_SOURCE[0] so the path resolves relative to the hook file
 # itself (two levels up from .claude/hooks/ to the workspace root),
 # not relative to the agent's cwd — the agent's cwd can be anywhere.
 # Deviation from brief: brief used ${_rules_dir%/.claude/hooks}/scripts/
 # which fails in the test environment (cwd → $WORK, no $WORK/scripts/).
 # shellcheck source=../../scripts/ws-k8s-guard.sh
-source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../scripts/ws-k8s-guard.sh" 2>/dev/null || true
+_k8s_guard_loaded=0
+if source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../scripts/ws-k8s-guard.sh" 2>/dev/null \
```

Shellcheck (0.11.0)

[info] Not following: ./../../scripts/ws-k8s-guard.sh was not specified as input (see shellcheck -x).

(SC1091)